src/bindings/python/CMakeLists.txt

```diff
@@ -411,3 +412,80 @@ if(OpenVINODeveloperPackage_FOUND)
 
     ov_cpack(${OV_CPACK_COMPONENTS_ALL})
 endif()
+
+#if(DEFINED ENV{CI} OR DEFINED ENV{TF_BUILD} OR DEFINED ENV{JENKINS_URL})
+if(TRUE)
+    #
+    # Generate and check .pyi stub files generated by pybind11-stubgen vs commited files
+    #
+    execute_process(COMMAND ${Python3_EXECUTABLE} -m pip show pybind11-stubgen
+                    OUTPUT_VARIABLE pybind11_stubgen_info OUTPUT_STRIP_TRAILING_WHITESPACE
+                    RESULT_VARIABLE pybind11_stubgen_result)
+
+    if(pybind11_stubgen_result)
+        message(FATAL_ERROR "pybind11-stubgen is not installed. Please install it using 'pip install pybind11-stubgen'.")
+    else()
+        string(REGEX MATCH "Version: ([\\.0-9]+)" pybind11_stubgen_version "${pybind11_stubgen_info}")
+        set(pybind11_stubgen_version ${CMAKE_MATCH_1})
+        message(STATUS "pybind11-stubgen version: ${pybind11_stubgen_version}")
+    endif()
+
+    if(OpenVINODeveloperPackage_FOUND)
```

Comment:
I suppose we don't need such check as python API cannot be built w/o dev package
See https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/CMakeLists.txt#L21

Reply:
Removed.